CMakeLists.txt

```diff
@@ -23,27 +23,6 @@ if(RE2C_REBUILD_PARSERS AND NOT RE2C_FOR_BUILD)
 endif()
 
 option(RE2C_REBUILD_DOCS "Regenerate manpage" OFF)
-if(RE2C_REBUILD_DOCS)
-    find_program(RST2MAN NAMES rst2man rst2man.py)
-    if(NOT RST2MAN)
-        message(FATAL_ERROR "rst2man (or rst2man.py) program is required for RE2C_REBUILD_DOCS")
-    endif()
```

Comment:
It should be possible to check for a python docutils package with something like this:
```
execute_process(
    COMMAND Python3::Interpreter -c "import docutils"
    RESULT_VARIABLE EXIT_CODE
    OUTPUT_QUIET
)
if (NOT ${EXIT_CODE} EQUAL 0)
    message(FATAL_ERROR "python package docutils (needed for docs) not found")
endif()
```
(Taken from https://stackoverflow.com/questions/62553007/cmake-check-python3-and-python-packages-presence.)

Reply:
Done.